README.md

```diff
@@ -1,12 +1,45 @@
-My thoughts on the takeaway challenge
+## Takeaway Challenge ##
+==================
+
+## How to Use
+
+```shell
+irb -r './lib/order.rb'
+```
+Create a new order, then choose any of the following methods:
+view_menu --> displays the menu
+select(dish, quantity) --> eg ("dishname", 3)
+place_order --> this submits your order
+
+eg.
```

Comment:
This is great! Makes it all super clear and easy to understand how to interact with the programme